Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -731,34 +731,21 @@ class DropDownMenu
     const anchor = document.createElement("a");
     anchor.setAttribute('role', 'menuitem');
     anchor.setAttribute('tabindex', '-1');
-    if (isEnabled)
+    if (isEnabled && itemInfo.Href != null)
     {
-      anchor.setAttribute('href', itemInfo.FallbackNavigationUrl);
+      anchor.setAttribute('href', itemInfo.Href);
+      if (itemInfo.Target != null)
+        anchor.setAttribute('target', itemInfo.Target);
```

Comment:
```suggestion
      anchor.href = itemInfo.Href;
      if (itemInfo.Target)
        anchor.target = itemInfo.Target;
```

Reply:
@a-ctor we have been using setAttribute in the rest of the code. Why change it here?

Reply:
I didn't notice the other `setAttribute` calls. It just doesn't look right to call `setAttribute` for something that is already exposed as a property on the element.

Reply:
@a-ctor symmetry with `removeAttribute`. And `removeAttribute` is removing the attribute in the DOM, setting it to empty string keeps the attribute.